scrapy/extensions/throttle.py

```diff
@@ -15,6 +18,12 @@ def __init__(self, crawler):
 
         self.debug = crawler.settings.getbool("AUTOTHROTTLE_DEBUG")
         self.target_concurrency = crawler.settings.getfloat("AUTOTHROTTLE_TARGET_CONCURRENCY")
+        self.enable_429 = crawler.settings.getbool("AUTOTHROTTLE_HANDLE_API_RATE_LIMIT")
```

Comment:
Based on similar settings, something in the line of `AUTOTHROTTLE_HANDLE_RATE_LIMIT_ENABLED` may be better. In any case, the internal variable (`enable_429`) should definitely not have 429 in it if we are going to allow a different status code to be considered a too-many-requests response.